setup.sh

```diff
@@ -1,56 +1,64 @@
 #!/bin/bash
-time=`date '+%Y%m%d_%H%M'`
-VAG_COMMON_DIR="/vagrant/"
+time=`TZ=Asia/Jerusalem date '+%Y%m%d_%H%M'`
+VAG_COMMON_DIR="/vagrant"
+HOME="/home/vagrant"
 LOG_DIR="$VAG_COMMON_DIR/log"
 OUTFILE="$LOG_DIR/${time}_vagrant-out.log"
+FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
+FLASK_DIR="$HOME/FlaskApp"
+FLASK_EXEC="$HOME/runFlask.sh"
+KILLFLASK_SCRIPT="$VAG_COMMON_DIR/killFlask.sh"
+PYTHON3="`which python3`"
+FLASK_ONBOOT=false # Currently Flask On Startup feature disabled [ Not Supported ]
 
 runSetup() {
     step=1
 
+    if [ ! -e $FLASK_APP_PY ]; then
+        echo "ERROR: Missing $FLASK_APP_PY file..." >> $OUTFILE
+        exit 1
+    else
+        chmod +x $FLASK_APP_PY
+    fi
+
+    if [ ! -d $FLASK_DIR ]; then
+        mkdir $FLASK_DIR
+    fi
+
     if [ ! -d $LOG_DIR ]; then
         mkdir $LOG_DIR
+    else
+        rm -f $LOG_DIR/*vagrant-out.log
```

Comment:
Do you really want to delete all previous log files?

Reply:
I wanted to avoid accumulation of logs and requirement of maintenance.
For some retention, I guess I could delete all previous to the last 4 logs for example.

Reply:
For application logs, you'll want to keep all of them with some sort of retention policy, which is probably out of scope for this. For installation logs, I would make sure to keep a few.
I would also suggest having a clean up script to deal with that and not build a retention policy inside the setup code.

I also don't think that we are up to that yet.

Reply:
Understood, so in order to keep the effort relevant I could redirect application log to a single log file in the meantime. And as mentioned will keep previous few for installation logs.

Should it be required, I could add a script for log retention (by size and number of previous logs) to be executed periodically via crontab.